topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
->   - ODV collection is created with the right template and opens on an empty global map.
->   - Click on the top left  : Import> ODV spreadsheet
->   - The “Select spreadsheet file(s)” popup window opens; Change “Files of Types” by selected “All Files(*)” and Select working “/Documents/ODV/galaxy/data/ODV tool output”
->   - The “Meta variable association” popup window opens. all meta variable are already selected > OK
->   - The “Data variable association” popup window opens. all data variable are already selected > OK
->   - Data set is now imported into the new collection
->
-> - 6l. Qualify the dataset if necessary. Go to the `<ODV user guide https://odv.awi.de/fileadmin/user_upload/odv/docs/ODV_guide.pdf>` for ODV description or follow the link to video.
+> ODV is a software for visualizing and/or qualifying scientific data.  
+> It can be used locally or directly on the Galaxy {% tool [Ocean Data View](interactive_tool_odv) %}.  
+{: .details}
+
+> <hands-on-title>Run ODV interactive tool</hands-on-title>
+> - Click on **Tools** or **Interactive Tools** in the left panel
+> - Search **ODV** interactive tool
+> - Select the tool. Its configuration page opens
+>
+> {% tool [Ocean Data View](interactive_tool_odv) %} with the following parameters:
+> - Ensure the tool version is **v5.8_1** (change via the 3 cubes on the left if needed)
+> - Set parameters:
+>   - *Load data automatically*: **Yes**
+>   - *NetCDF or tabular text file*: select `ODV collection manager output` or drag and drop `odv_collection.txt`
+>   - *Do you have a view?*: **Yes** → drag and drop `qualification_startingPoint_nitrate.xview`
+> - Click **Run**
+>
+> ![ODV launch](../../images/bgc_calib/odv_launch.png)
 {: .hands_on}
 
-> <hands-on-title>Export history once qualification is finished</hands-on-title>
-> WARNING : Be sure that all filters or zoom windows are relaxed.
+> <details-title>How to open ODV</details-title>
+> - When ODV is ready, a red dot appears on **Interactive Tools**  
+> - Click it → open the ODV panel → click the expand symbol  
+> - ODV opens in a new window with your selected view  
 >
-> 7a. To export history information, click on the top left : 
->   - Export > History
->   - the “Export station history” opens; Select “working/Documents/ODV/galaxy/outputs
->   - Change the name of the history if you want > Save > OK > OK
->   - close ODV : File > exit
+> ![ODV interactive tool](../../images/bgc_calib/odv_launch_2.png)
+> ![ODV nitrate view](../../images/bgc_calib/odv_view.png)
+{: .details}
+
+> <tip-title>ODV inactive</tip-title>
+> After some inactivity, the ODV window may close. Just refresh the browser tab.
+{: .tip}
+
+> <tip-title>Copy/paste in ODV</tip-title>
+> Open the clipboard (3 dots on the left panel) to copy-paste text.
+> ![Clipboard in ODV](../../images/coastal_water_dyn/clipboard.png)
+{: .tip}
+
+### Step 3: Qualify the dataset with ODV

```

Comment:
Please remove the Step 1/2/3 etc, it isnt necessary and just makes it harder to add steps later